ex4/readme.md

```diff
@@ -46,7 +46,7 @@ export default {
     }
 }
 ```
-In both `JSONTableDelegate` and `JSONFilterBarDelegate` files, import and add the new `JSONBaseDelegate` to the assign call as a third argument.
```

Comment:
It is actually the 'assign' function call. Maybe add some backticks to make it clear, that the reference is to the code.